diff -uraN mauikit-imagetools-v2.1.0/CMakeLists.txt omv-mauikit-imagetools-v2.1.0/CMakeLists.txt
--- mauikit-imagetools-v2.1.0/CMakeLists.txt	2021-10-26 00:54:32.000000000 +0200
+++ omv-mauikit-imagetools-v2.1.0/CMakeLists.txt	2021-12-29 03:45:36.740250101 +0100
@@ -75,7 +75,7 @@
         COMPONENT Devel)
 
 find_package(Qt5 ${REQUIRED_QT_VERSION} COMPONENTS Core Quick Qml Sql Positioning REQUIRED)
-find_package(LibExiv2 0.21 REQUIRED)
+find_package(Exiv2 0.21 REQUIRED)
 
 if(ANDROID)
 find_package(Qt5 ${REQUIRED_QT_VERSION} COMPONENTS AndroidExtras)
